hack/local-up-kubeedge.sh

```diff
@@ -31,6 +31,19 @@ fi
 export CLUSTER_CONTEXT="--name ${CLUSTER_NAME}"
 
 function install_cr() {
+
+  if [[ "${CONTAINER_RUNTIME}" = "containerd" ]]; then
+      echo "No need to download container runtime for containerd"
+      return
+  fi
+
+  SKIP_CR_INSTALL=${SKIP_CR_INSTALL:-"false"}
+  if [[ "${SKIP_CR_INSTALL}" = "true" ]]; then
+    echo "Skip container runtime installation"
+    return
+  fi
```

Comment:
How about making the SKIP_CR_INSTALL as global variable?
  
  if [[ "${SKIP_CR_INSTALL}" = "false" ]]; then
    install_cr
  else
    echo "Skip container runtime installation"
  fi

Reply:
follow the comment, 

move the `SKIP_CR_INSTALL` env out of the `install_cr` function.